Week-9/todoapplication/src/TodoPage.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "./AppWideCSS.css";
+import { Header, Content } from "./components/index";
+
+const TodoPage = (props) => {
```

Comment:
What is the use of props if you aren't using it ?

Reply:
Addressed this issue. Removed it.